packages/date-picker/src/_subcomponents/Calendar/LegacyCalendarRange.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { DayPicker, DateRange } from "react-day-picker"
+import { DayClickEventHandler } from "react-day-picker/dist/types/EventHandlers"
+import { Matcher } from "react-day-picker/dist/types/Matchers"
+import { DayOfWeek } from "../../enums"
+import { isInvalidDate } from "../../utils/isInvalidDate"
+import { IconLeft } from "./components/IconLeft"
+import { IconRight } from "./components/IconRight"
+import { isValidWeekStartsOn } from "./utils/isValidWeekStartsOn"
+import { legacyCalendarRangeClasses } from "./CalendarClasses"
+
+export type LegacyCalendarRangeElement = HTMLDivElement
+
+export type LegacyCalendarRangeProps = {
+  id: string
+  classNameOverride?: string
+  defaultMonth?: Date
+  weekStartsOn?: DayOfWeek
+  disabledDays?: Matcher[]
+  selectedRange?: DateRange
+  locale: Locale
+  onDayChange: DayClickEventHandler
+}
+
+export const LegacyCalendarRange: React.VFC<LegacyCalendarRangeProps> = ({
```

Comment:
This looks largely the same as your new `CalendarRange` why did it get set as `Legacy*`?